website/www/site/content/en/blog/beam-2.56.0.md

```diff
@@ -30,11 +30,13 @@ For more information on changes in 2.56.0, check out the [detailed release notes
 ## Highlights
 
 * Added FlinkRunner for Flink 1.17, removed support for Flink 1.12 and 1.13. Previous version of Pipeline running on Flink 1.16 and below can be upgraded to 1.17, if the Pipeline is first updated to Beam 2.56.0 with the same Flink version. After Pipeline runs with Beam 2.56.0, it should be possible to upgrade to FlinkRunner with Flink 1.17. ([#29939](https://github.com/apache/beam/issues/29939))
+* New Managed I/O Java API ([#30830](https://github.com/apache/beam/pull/30830)).
```

Comment:
to sync, can you also update changes.md?

Reply:
Done.